catalog/v4.19/security-profiles-operator/catalog.yaml

```diff
@@ -26,1565 +29,1937 @@ image: registry.redhat.io/compliance/openshift-security-profiles-operator-bundle
+        operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
+      apiServiceDefinitions: {}
+      crdDescriptions:
+        owned:
+          - description: ProfileBinding is the Schema for the profilebindings API.
+            displayName: Profile Binding
+            kind: ProfileBinding
+            name: profilebindings.security-profiles-operator.x-k8s.io
+            version: v1alpha1
+          - description: ProfileRecording is the Schema for the profilerecordings API.
+            displayName: Profile Recording
+            kind: ProfileRecording
+            name: profilerecordings.security-profiles-operator.x-k8s.io
+            version: v1alpha1
+          - description: SeccompProfile is a cluster level specification for a seccomp profile. See https://github.com/opencontainers/runtime-spec/blob/master/config-linux.md#seccomp
+            displayName: Seccomp Profile
+            kind: SeccompProfile
+            name: seccompprofiles.security-profiles-operator.x-k8s.io
+            version: v1beta1
+          - description: SecurityProfileNodeStatus is a per-node status of a security profile
+            displayName: Security Profile Node Status
+            kind: SecurityProfileNodeStatus
+            name: securityprofilenodestatuses.security-profiles-operator.x-k8s.io
+            version: v1alpha1
+          - description: SecurityProfilesOperatorDaemon is the Schema to configure the spod deployment.
+            displayName: Security Profiles Operator Daemon
+            kind: SecurityProfilesOperatorDaemon
+            name: securityprofilesoperatordaemons.security-profiles-operator.x-k8s.io
+            version: v1alpha1
+          - kind: RawSelinuxProfile
+            name: rawselinuxprofiles.security-profiles-operator.x-k8s.io
+            version: v1alpha2
+          - description: SelinuxProfile is the Schema for the selinuxprofiles API.
+            displayName: Selinux Profile
+            kind: SelinuxProfile
+            name: selinuxprofiles.security-profiles-operator.x-k8s.io
+            version: v1alpha2
+          - description: AppArmorProfile is a cluster level specification for an AppArmor profile.
+            displayName: App Armor Profile
+            kind: AppArmorProfile
+            name: apparmorprofiles.security-profiles-operator.x-k8s.io
+            version: v1alpha1
+      description: SPO is an operator which aims to make it easier for users to use SELinux, seccomp and AppArmor in Kubernetes clusters
+      displayName: Security Profiles Operator
+      installModes:
+        - supported: true
+          type: OwnNamespace
+        - supported: true
+          type: SingleNamespace
+        - supported: true
+          type: MultiNamespace
+        - supported: true
+          type: AllNamespaces
+      keywords:
+        - security
+        - selinux
+        - seccomp
+        - apparmor
+        - ebpf
+      labels:
+        operatorframework.io/arch.amd64: "true"
```

Comment:
```suggestion
        operatorframework.io/arch.amd64: "true"
        operatorframework.io/arch.ppc64le: "true"
```

Reply:
Missing the ppc64le architecture

Reply:
Proper value is 'supported'

Reply:
those are from old release catalog, the latest v0.9.0 should already have the correct `supported` label